packages/react/src/components/editor/JSONEditorConnected.tsx

```diff
@@ -1,22 +1,27 @@
-import {connect} from 'react-redux';
+import {useDispatch, useSelector} from 'react-redux';
+import React, {useEffect} from 'react';
 
-import {PlasmaState} from '../../PlasmaState';
 import {IDispatch} from '../../utils';
 import {JSONEditor, JSONEditorDispatchProps, JSONEditorProps, JSONEditorStateProps} from './JSONEditor';
 import {JSONEditorActions} from './JSONEditorActions';
 import {JSONEditorSelectors} from './JSONEditorSelectors';
 
-const mapStateToProps = (state: PlasmaState, ownProps: JSONEditorProps): JSONEditorStateProps => ({
-    value: JSONEditorSelectors.getValue(state, ownProps.id),
-});
+export const JSONEditorConnected = (props: JSONEditorStateProps & JSONEditorDispatchProps & JSONEditorProps) => {
```

Comment:
I don't think you need `JSONEditorStateProps & JSONEditorDispatchProps`, previously those weren't exposed by the connected component

```suggestion
export const JSONEditorConnected = (props: JSONEditorProps) => {
```